clients/drcachesim/reader/reader.cpp

```diff
@@ -79,7 +79,7 @@ reader_t::operator++()
             // We've already presented the thread exit entry to the analyzer.
             continue;
         }
-        VPRINT(this, 4, "RECV: type=%s (%d), size=%d, addr=0x%zx\n",
+        VPRINT(this, 5, "RECV: type=%s (%d), size=%d, addr=0x%zx\n",
```

Comment:
In the PR description:

>  when the scheduler has read ahed.

s/ahed/ahead/

Reply:
Done.